Flaky test - TestCreateAccountAndLogin

```
INFO [01-04|14:21:27.743|github.com/waku-org/go-waku/waku/v2/peermanager/peer_manager.go:388]               adding peer to peerstore                 peer=16Uiu2HAm1fVVprL9EaDpDw4frNX3CPfZu5cBqhtk9Ncm6WCvprpv
    create_account_and_login_test.go:49: TestCreateAccountAndLogin: create account user1 and login successfully
    create_account_and_login_test.go:53: TestCreateAccountAndLogin: logouting
panic: test timed out after 30m0s
```

reproduced in this [PR](https://github.com/status-im/status-go/pull/4969) test `TestMessageSent` 

```
    messenger_raw_message_resend_test.go:110: Boot Node Stop completed successfully.
    messenger_raw_message_resend_test.go:110: Bob Logout completed successfully.
    messenger_raw_message_resend_test.go:117: Timeout occurred, the following operations did not complete in time:
    messenger_raw_message_resend_test.go:120: Alice Logout is still pending.
    --- FAIL: TestMessengerRawMessageResendTestSuite/TestMessageSent (34.73s)
```

--- a/go.mod
+++ b/go.mod
@@ -90,7 +90,7 @@ require (
 	github.com/schollz/peerdiscovery v1.7.0
 	github.com/siphiuel/lc-proxy-wrapper v0.0.0-20230516150924-246507cee8c7
 	github.com/urfave/cli/v2 v2.24.4
-	github.com/waku-org/go-waku v0.8.1-0.20240322182925-dd81e1d46971
+	github.com/waku-org/go-waku v0.8.1-0.20240415131212-6d889ca3e2fe
 	github.com/wk8/go-ordered-map/v2 v2.1.7
 	github.com/yeqown/go-qrcode/v2 v2.2.1
 	github.com/yeqown/go-qrcode/writer/standard v1.2.1

